lib/solidus_importer/base_logger.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  class BaseLogger
+    delegate :debug, :error, :fatal, :info, :unknown, :warn, to: :logger_engine
```

Comment:
`::Logger` can be used directly if we're just delegating those methods, I guess this is a leftover of a more complex implementation that is now gone.